### DISCUSSION TITLE
part-ii/2019-5/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#114] [Question comments]
Part II 2019 Algebraic Topology: Paper 4, Section II, F 

https://questions.tripos.org/part-ii/2019-5/

**Comment by ArthurConmy** [comment]
I couldn't do this at first, but then saw 2014 P4L which is basically the same thing with handholding on how to do the last part.

Homology groups are $H_0 = \mathbb{ Z}, H_1 = \mathbb{ Z}, H_2 =\mathbb{ Z}^n$.

i) $\phi$ acts trivially on $H_0$ and $H_1$ since $\phi^3 = \text{Id}$. So $\text{tr}(f_2)=0$. Since $f_2^3 = \text{Id}$, it's diagonalisable with eigenvalues $1, \omega, \omega^2$, suppose there are $A, B$ and $C$ of these values. So $A+B+C=n, B=C, A-2B=0$ (because $f_2$ is a map of $n$D spaces, trace is real, and trace is 0 respectively) which rearrange to give $3|n$.

ii) we can have $\phi_1 = \pm 1$ here, but since $\phi_2^2 = \text{Id}$ there are only eigenvalues $\pm 1$ and hence the same "trace" and "number of eigenvalues = $n$" equations give $2|n$.

**Reply by ArthurConmy** [reply]
The lesson here is to work with $H_i(X)$ and NOT make things complicated thinking about simplicial approximations $H_i(K^{(r)})$ (we lose control of the "homeomorphism" constraint), which cost me time ...
